CHANGES.txt

```diff
@@ -5,7 +5,9 @@ CHANGES
 3.0.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Added support for Python 3.4, 3.5 and 3.6.
+
+- Dropped support for Python 2.6 and 3.3.
```

Comment:
As this change is backwards incompatible the version number should be increased to `4.0` (here and in `setup.py`.)

Reply:
Done!